test/functional/test_nvme_hotplug.py

```diff
@@ -0,0 +1,272 @@
+"""
+*********************************************************
+Copyright @ 2015 EMC Corporation All Rights Reserved
+*********************************************************
+"""
+import unittest
+import os
+import json
+import time
+from infrasim import model
+from infrasim import helper
+from infrasim.helper import UnixSocket
+from infrasim import sshclient
+from test import fixtures
+
+
+old_path = os.environ.get('PATH')
+new_path = '{}/bin:{}'.format(os.environ.get('PYTHONPATH'), old_path)
+conf = {}
+
+
+def setup_module():
+    os.environ['PATH'] = new_path
+
+
+def teardown_module():
+    os.environ['PATH'] = old_path
+
+
+class test_drive_insert(unittest.TestCase):
+
+    @staticmethod
+    def start_node():
+        global conf
+        global path
+        nvme_config = fixtures.NvmeConfig()
+        conf = nvme_config.get_node_info()
+        conf["compute"]["boot"] = {
+            "boot_order": "c"
+        }
+        conf["compute"]["storage_backend"] = [
+            {
+                "type": "ahci",
+                "max_drive_per_controller": 6,
+                "drives": [
+                    {
+                        "size": 40,
+                        "model": "SATADOM",
+                        "serial": "20160518AA851134100",
+                        "file": fixtures.image
+                    }
+                ]
+            },
+            {
+                "cmb_size_mb": 1,
+                "drives": [
```

Comment:
wrong definition of nvme file. there is no "drives" field.
all field in same level.